tests/integration/testlib/assertlib.py

```diff
@@ -76,6 +77,7 @@ def _prepare_state_for_verify(desired_state_data):
     desired_state_data = copy.deepcopy(desired_state_data)
     desired_state_data.pop(Route.KEY, None)
     desired_state_data.pop(DNS.KEY, None)
+    desired_state_data.pop(RouteRule.KEY, None)
```

Comment:
Here is an example where comments rot. Please either update the comment or remove it and move these calls to a helper function that is named appropriately.